src/fixes/duplicated_key.rs

```diff
@@ -0,0 +1,152 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct DuplicatedKeyFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for DuplicatedKeyFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "DuplicatedKey",
+        }
+    }
+}
+
+impl Fix for DuplicatedKeyFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_warnings(
+        &self,
+        warnings: Vec<&mut Warning>,
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let len = warnings.len();
+
+        for warning in warnings {
+            let line = lines
+                .get_mut(warning.line_number() - 1)
+                .expect("warning references inexistent line");
+
+            line.raw_string = format!("#{}", line.trimmed_string());
```

Comment:
I'm wondering if it's necessary to do ```.trimmed_string``` here - presumably if there is extra whitespace on a line, that should be fixed by another Fixer? (I'm not sure about this, so if there's a good reason I'd like to hear why)